src/views/Sjofn/Homework4.vue

```diff
@@ -0,0 +1,166 @@
+        {
+          'nr': 4,
+          'name': 'Sweet pea',
+          'image': 'sweetpea.jpg',
+          'description': 'Lathyrus odoratus is a flowering plant in the genus Lathyrus in the family Fabaceae (legumes), native to Sicily, Cyprus, southern Italy and the Aegean Islands. It is an annual climbing plant, growing to a height of 1–2 metres (3 ft 3 in–6 ft 7 in), where suitable support is available. The leaves are pinnate with two leaflets and a terminal tendril, which twines around supporting plants and structures, helping the sweet pea to climb. In the wild plant the flowers are purple, 2–3.5 centimetres (0.79–1.38 in) broad; they are larger and very variable in color in the many cultivars.'
+        },
+        {
+          'nr': 5,
+          'name': 'Crownvetch',
+          'image': 'crownvetch.jpg',
+          'description': 'Securigera varia (synonym Coronilla varia), commonly known as crownvetch[2] or purple crown vetch, is a low-growing legume vine. It is native to Africa, Asia and Europe and is commonly used throughout the United States and Canada for erosion control, roadside planting and soil rehabilitation. It has become an invasive species in many states of the US. Crownvetch grows 1 to 2 feet tall and bears small clusters of 1/2-inch pink and white flowers from early summer to late fall. The variety Penngift produces mounds of pink flowers. Crownvetch is a tough, aggressive spreading plant that will crowd out its neighbors in a show garden but is well suited to a sunny bank, where it will grow for decades with little to no fertilizing, mowing, or weeding since the thick foliage prevents weeds from growing amongst it. Its deep, tenacious, complex root system  and thick, fern-like leaves provide excellent erosion control where it is used as a ground cover. However, this legume has a very long germination period of about 6 months and does not create full coverage until two or three years later. Crownvetch is toxic to horses and other non-ruminants because of the presence of nitroglycosides. If consumed in large amounts, it can cause slow growth, paralysis, or even death. However, this is not true for ruminant animals such as cattle, goats, and sheep. These aliphatic nitro compounds are degraded in ruminant digestion and do not affect the animals.'
+        }
+      ],
+      bool: '',
+      adjective: '',
+      characterName: '',
+      name: '',
+      age: '',
+      gender: '',
+      eyeColor: '',
+      hobbies: '',
+      a: '',
+      b: ''
+    }
+  },
+  methods: {
+    negate (bool) {
+      // return !bool
+      console.log(!(bool === 'true'))
+    },
+
+    printTitle (adjective, characterName) {
+      console.log(`${adjective} adventures of ${characterName}`)
+    },
+
+    buildUserModel () {
+      let user = {
+        'name': this.name,
+        'age': this.age,
+        'gender': this.gender,
+        'eyeColor': this.eyeColor,
+        'hobbies': this.hobbies
+      }
+      console.log(user)
```

Comment:
You can avoid creating extra variable here and print the value directly:
`console.log({
   'name': this.name,
   'age': this.age,
    'gender': this.gender,
    'eyeColor': this.eyeColor,
    'hobbies': this.hobbies
})`